### PR TITLE
Fix stronghold backup/restore

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Fix stronghold backup/restore

## Links to any relevant issues

Fixes https://github.com/iotaledger/iota-sdk/issues/2232

## Type of change

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

Test

### CODE REVIEW

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
Conflicts

File: bindings/nodejs/CHANGELOG.md [hunk 1]
```diff
@@ -23,7 +23,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Fixed
 
+
```
Comment (Thoralf-M) [inline bindings/nodejs/CHANGELOG.md]:
```suggestion
```

Comment (thibault-martinez) [inline bindings/nodejs/CHANGELOG.md]: :trollface:

Comment (thibault-martinez) [reply bindings/nodejs/CHANGELOG.md]: too fast

Comment (Thoralf-M) [reply bindings/nodejs/CHANGELOG.md]: Also just saw this

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**